e2e/js/js-cdn/src/app/app.element.ts

```diff
@@ -0,0 +1,450 @@
+import './app.element.css'
+
+const SDK_KEY =
+    process.env.DEVCYCLE_CLIENT_SDK_KEY || '<DEVCYCLE_CLIENT_SDK_KEY>'
+export class AppElement extends HTMLElement {
+    constructor() {
+        super()
+    }
+
+    public static observedAttributes = []
+
+    updateInnerHTML(): void {
+        const enabledVariable = devcycleClient.variableValue(
+            'enabled-feature',
+            false,
+        )
+
+        const disabledVariable = devcycleClient.variableValue(
+            'disabled-feature',
+            true,
+        )
+
+        const defaultVariable = devcycleClient.variableValue(
+            'default-feature',
+            true,
+        )
+        this.innerHTML = `
+          <div class="wrapper">
+            <div class="container">
+              <!--  WELCOME  -->
+              <div id="welcome">
+                <h1>
+                  <span> Hello there </span>
+                </h1>
+              </div>
+              <div id="variableKey">
+                <h1>
+                  <span> JS Web Elements</span>
+                </h1>
+              </div>
+              <div id="enabledVariableKey">
+              <h1>
+                <span>variable enabled-feature = </span>
+                ${enabledVariable}
+              </h1>
+            </div>
+              <div id="disabledVariableKey">
+                <h1>
+                  <span>variable disabled-feature = </span>
+                  ${disabledVariable}
+                </h1>
+              </div>
+              <div id="defaultVariableKey">
+                <h1>
+                  <span>variable default-feature = </span>
+                  ${defaultVariable}
+                </h1>
+              </div>
+              <!--  HERO  -->
+              <div id="hero" class="rounded">
```

Comment:
do we need all this extra stuff?

Reply:
nah, Ill cut it. Just auto-generated tings